packages/governance/src/paramGovernance/assertions.js

```diff
@@ -0,0 +1,64 @@
+// @ts-check
+
+import { isRemotable } from '@agoric/marshal';
+import { assertIsRatio } from '@agoric/zoe/src/contractSupport/ratio.js';
+
+const { details: X } = assert;
+
+const makeAssertBrand = name => {
+  return brand => {
+    assert(
+      // @ts-ignore value is undifferentiated to this point
+      isRemotable(brand),
+      X`value for ${name} must be a brand, was ${brand}`,
+    );
+  };
+};
```

Comment:
I was going to suggest `@returns { asserts brand is Brand }` but this code doesn't actually determine that the argument is a brand. Is this package-internal or is it exported to contract writers? If it's exported, call it `looksLikeBrand` or some such? Maybe even if it's not exported?

Likewise `makeAssertInstallation `, `makeAssertInstallation`, though it looks like we have plans for those to actually correctly judge their arguments.